common/scripted_triggers/wc_portrait_triggers.txt

```diff
@@ -26,6 +26,10 @@ has_no_clothes_trigger = {
 	has_quilboar_portrait_trigger = no
 }
 
+prefer_no_headgear_culture_trigger = {
+	scope:culture = { has_graphical_culture = no_headgear_culture_gfx }
+}
```

Comment:
Please see the last [commit](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/852/commits/7e6a49b7ed5767153ffbbe163d356a9e5a01b26d) for reference. `no_headgear_gfx` was used prior to this commit but I decided to create a new graphical gfx because `no_headgear_gfx` relies on the trigger `prefer_no_lowborn_headgear` which has a condition `duke_or_highter_clothing_trigger = no` (line 42/46 of `common/scripted_triggers/wc_portrait_triggers.txt`).

This present two problems: 
1) Uneven application (Vanilla code blocks like in \portrait_modifiers`01_clothes.txt. do not use this trigger and since I deleted my unique dwarven code block in the previous commits, `portrait_dwarven_clothing_trigger` is now inserted into vanilla code blocks without it)

2) Undesired effect
Dukes and higher still wear their headgear during war.

There is possibly a way around this (perhaps you can suggest since I am a noob) by modifying the existing `prefer_no_lowborn_headgear` trigger but I felt it was cleaner to have a separate base trigger specifically for use in the cases of other cultures like Wildhammer that do not use battle headgear. 

> Better don't refer to scope:culture. It may not exist.
I can test it after removing. Currently it does work however.

Will also shorten name.

Reply:
> Vanilla code blocks like in \portrait_modifiers01_clothes.txt. do not use this trigger and since I deleted my unique dwarven code block in the previous commits, portrait_dwarven_clothing_trigger` is now inserted into vanilla code blocks without it.

`no_headgear_gfx` not used anywhere except `prefer_no_lowborn_headgear`.

> My initial attempted workaround was highest_held_title_tier <= tier_duchy

Why do you need this trigger at all then? Use `prefer_no_lowborn_headgear` and `prefer_no_crown`.
Or keep the trigger and rename it to clarify what exactly you want.

Reply:
Resolved by merging effects to `prefer_no_crown`